### PR TITLE
.travis.yml: Also run examples in Neovim mode.

### LINKED ISSUE #64: Run vroom examples under neovim-mode in Travis CI
It will be awesome to get integration test coverage for neovim in Travis CI. I tried enabling setting it up to run the examples dir in neovim mode, but ran into this error:

```
ERROR: Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.8/lib/python2.7/site-packages/vroom/runner.py", line 70, in __call__
    self.env.vim.Start()
  File "/home/travis/virtualenv/python2.7.8/lib/python2.7/site-packages/vroom/neovim_mod.py", line 44, in Start
    session = neovim.socket_session(self.args.servername)
  File "/home/travis/virtualenv/python2.7.8/lib/python2.7/site-packages/neovim/msgpack_rpc/__init__.py", line 31, in socket_session
    return session('socket', path)
  File "/home/travis/virtualenv/python2.7.8/lib/python2.7/site-packages/neovim/msgpack_rpc/__init__.py", line 17, in session
    loop = EventLoop(transport_type, *args, **kwargs)
  File "/home/travis/virtualenv/python2.7.8/lib/python2.7/site-packages/neovim/msgpack_rpc/event_loop/base.py", line 87, in __init__
    getattr(self, '_connect_{0}'.format(transport_type))(*args)
  File "/home/travis/virtualenv/python2.7.8/lib/python2.7/site-packages/neovim/msgpack_rpc/event_loop/asyncio.py", line 84, in _connect_socket
    self._loop.run_until_complete(coroutine)
  File "/home/travis/virtualenv/python2.7.8/lib/python2.7/site-packages/trollius/base_events.py", line 281, in run_until_complete
    return future.result()
  File "/home/travis/virtualenv/python2.7.8/lib/python2.7/site-packages/trollius/futures.py", line 277, in result
    raise self._exception
FileNotFoundError: [Errno 2] No such file or directory
```

### PR DESCRIPTION
**Pull request by fwalch**
Fixes #64.

### CODE REVIEW

**Comment by fwalch** [comment]
@dbarnett regarding https://github.com/google/vroom/pull/75#discussion_r19325702, I found out that this could be written like:

```yaml
before_install: |
    if [ $CI_TARGET = neovim ]; then
      pip install neovim
    fi
```

source: http://stackoverflow.com/a/15365296/249642

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
 python:
-  - "3.4"
+  - 3.4
+env:
+  matrix:
+    - CI_TARGET=vim
+    - CI_TARGET=neovim VROOM_ARGS=--neovim
+before_install:
+  - if [ $CI_TARGET = neovim ]; then
+      pip install neovim;
```
Comment (lpil) [inline .travis.yml]:
```yaml
before_install: |
  this
  is
  multiline
```

Comment (fwalch) [inline .travis.yml]: @lpil see https://github.com/google/vroom/pull/75#issuecomment-70483028

Comment (lpil) [inline .travis.yml]: Ah, cool. I found this commit outside the context of the pull request :)

**Pull request merged**